### ISSUE TITLE
Remove alpha tag from Grid component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2874]
**Is your feature request related to a problem? Please describe.**
The Grid component is currently an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after a successful a11y swarm on the component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
